pkgs/by-name/gi/gitlab-timelogs/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  darwin,
+  fetchCrate,
+  iconv,
+  lib,
+  openssl,
+  pkg-config,
+  rustPlatform,
+  stdenv,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "gitlab-timelogs";
+  version = "0.3.0";
+
+  src = fetchCrate {
+    inherit pname version;
+    hash = "sha256-n+Jvm4RqHkXIeQcY55iOEBgwvbr77vLMhqxXgdau5MQ=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs =
+    [ openssl ]
+    ++ lib.optionals stdenv.isDarwin [
+      darwin.apple_sdk.frameworks.SystemConfiguration
+      iconv
+    ];
+
+  cargoHash = "sha256-REE7DWY0l4TTDTwWFWVr3Zk/oLQlOjrbFEWSFUlBEig=";
+
+  meta = {
+    description = " CLI utility to support you with your time logs in GitLab";
+    mainProgram = "ttfb";
```

Comment:
Shouldn't this be `gitlab-timelogs`?

Reply:
yikes, copy & paste

Reply:
fyi: There is no need of this as the binary name matches the derivation name.